src/applications/journal_display/journal_controller.c

```diff
@@ -207,10 +208,10 @@ static void JournalController_SetupBgs(BgConfig *bgConfig)
         0
     };
 
-    Bg_InitFromTemplate(bgConfig, 0, &v1, 0);
+    Bg_InitFromTemplate(bgConfig, 0, &bgTemplate1, 0);
     Bg_ClearTilemap(bgConfig, 0);
 
-    BgTemplate v2 = {
+    BgTemplate bgTemplate2 = {
```

Comment:
**chore**: Not required, but it would be great to use C99 struct-initializers for these declarations. Refer to `SetupBgs` in `src/applications/options_menu.c` for an example.